pkg/translate/output.go

```diff
@@ -111,41 +111,69 @@ func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
 	return strconv.Itoa(int(bs))
 }
 
-func toBlockTopology(wr io.Writer, root *topology.Vertex) error {
-	// traverse tree topology and when a node is reached, check within blockRoot for domain and print that domain.
+/*
```

Comment:
Do we need to keep the commented-out code?

Reply:
No, we do not. Just for review purposes. I can remove it once everyone got a chance to review.